adapters/oidc/js/dist/keycloak.d.ts

```diff
@@ -627,11 +627,11 @@ declare class Keycloak {
 	hasRealmRole(role: string): boolean;
 
 	/**
-	* Returns true if the token has the given role for the resource.
-	* @param role A role name.
+	* Returns true if the token has the given role(or at least one role for multiple roles) for the resource.
+	* @param role A role(s) name.
 	* @param resource If not specified, `clientId` is used.
 	*/
-	hasResourceRole(role: string, resource?: string): boolean;
+	hasResourceRole(role: string | string[], resource?: string): boolean;

```

Comment:
Hmmm, taking a closer look at this change I am left wondering if it's really needed. If the consumer of this would like to check an array one could simply do the following:

```ts
const hasRole = roles.some(role => keycloak.hasResourceRole(role))
```

Apologies for the back an forth, but I think I'd rather keep the implementation of Keycloak JS itself unchanged. Could you remove this change from it?

Reply:
> Hmmm, taking a closer look at this change I am left wondering if it's really needed. If the consumer of this would like to check an array one could simply do the following:
> 
> ```ts
> const hasRole = roles.some(role => keycloak.hasResourceRole(role))
> ```
> 
> Apologies for the back an forth, but I think I'd rather keep the implementation of Keycloak JS itself unchanged. Could you remove this change from it?

@jonkoops I agree that it is more clear. My only thought was that `keycloak.hasResourceRole` will be call 2-3 times even if user has no account roles. However, this is rare. So I will procceed with requested change soon.